src/Tempest/Console/src/Input/ConsoleArgumentDefinition.php

```diff
@@ -24,18 +24,17 @@ public function __construct(
     public static function fromParameter(ParameterReflector $parameter): ConsoleArgumentDefinition
     {
         $attribute = $parameter->getAttribute(ConsoleArgument::class);
-
         $type = $parameter->getType();
 
         return new ConsoleArgumentDefinition(
-            name: $parameter->getName(),
-            type: $type->getName(),
-            default: $parameter->isDefaultValueAvailable() ? $parameter->getDefaultValue() : null,
-            hasDefault: $parameter->isDefaultValueAvailable(),
-            position: $parameter->getPosition(),
+            name       : $attribute?->name ?? $parameter->getName(),
```

Comment:
We don't use colon alignment, but apparently PHP CS fixer doesn't fix it :/

Reply:
Ok good to know
I'll fix it ASAP 👍

Reply:
done